src/gui/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ transferlistsortmodel.h
 transferlistwidget.h
 updownratiodialog.h
 utils.h
+uithememanager.h
```

Comment:
wrong order here and at other places too.

Reply:
@jagannatharjun:  It is still in the wrong place.  These filenames should be listed in **alphabetical order**.  

In the English version of the Latin alphabet, `i` comes before `p`.  

```
updownratiodialog.h
uithememanager.h
```

…should be… 

```
uithememanager.h
updownratiodialog.h
```